tests/test_integration.py

```diff
@@ -343,17 +349,19 @@ def test_interpreter_constraints_to_pex_info_py2():
     assert set(['>=2.7', '<3']) == set(pex_info.interpreter_constraints)
 
 
-@pytest.mark.skip('https://github.com/pantsbuild/pex/issues/470')
 def test_interpreter_constraints_to_pex_info_py3():
```

Comment:
`test_pex_python` in `test_integration.py` needs the same treatment as this test - both were disabled.